integration-tests/src/main/resources/META-INF/services/tachyon.underfs.UnderFileSystemFactory

```diff
@@ -0,0 +1,2 @@
+tachyon.underfs.hdfs.HdfsUnderFileSystemFactory
+tachyon.underfs.glusterfs.GlusterFSUnderFileSystemFactory
```

Comment:
Why are these entries necessary? I thought these should be already be included in the underfs submodules?